### PR TITLE
Add unsteady CHT tutorial files.

### PR DESCRIPTION
**Pull request by TobiKattmann**
Goes along with su2code/su2code.github.io/pull/26. 

5 cfg's and 1 mesh file for one cht unsteady case

### CODE REVIEW

**Comment by oleburghardt** [comment]
Thanks. Please hold on a second with this one, some config options in those files are to be replaced in https://github.com/su2code/SU2/pull/1226 :)

File: multiphysics/unsteady_cht/solid_cylinder1.cfg [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+SOLID_DENSITY= 0.0210322
+%MATERIAL_DENSITY= 0.0210322
+%
+% Solid specific heat (J/kg*K)
+SPECIFIC_HEAT_CP= 1004.703
+%
+% Solid thermal conductivity (W/m*K)
+SOLID_THERMAL_CONDUCTIVITY= 0.1028
+%KT_CONSTANT= 0.1028
+%
```
Comment (TobiKattmann) [inline multiphysics/unsteady_cht/solid_cylinder1.cfg]: I already prepared here the cfg option changes for your PR @oleburghardt

File: multiphysics/unsteady_cht/flow_cylinder.cfg [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+%
+% Linael solver ILU preconditioner fill-in level (0 by default)
+LINEAR_SOLVER_ILU_FILL_IN= 0
+%
+% Minimum error of the linear solver for implicit formulations
+LINEAR_SOLVER_ERROR= 1E-15
+%
+% Max number of iterations of the linear solver for the implicit formulation
+LINEAR_SOLVER_ITER= 20
+%
```
Comment (TobiKattmann) [inline multiphysics/unsteady_cht/flow_cylinder.cfg:117]: this was 5 iterations before, with this setup you get a vortex shedding, with 5 one doesn't

File: multiphysics/unsteady_cht/solid_cylinder1.cfg (hunk 1 above)
Comment (TobiKattmann) [reply multiphysics/unsteady_cht/solid_cylinder1.cfg]: ok i already changed them ahead

**Comment by TobiKattmann** [comment]
This is good to go from my point of view. @oleburghardt there is no reason imo to hold back here for su2code/su2/pull/1226 as these cfg are not yet included in a testcase. I want to move the regression test case to tutorials.py and use these cfgs. If ok i could sneak them in your PR for that. Then it also makes sense to merge this now as otherwise those options would fail...

**Pull request merged**